cfgov/v1/models/images.py

```diff
@@ -87,6 +87,14 @@ def default_alt_text(self):
         # Override Wagtail default of setting alt text to the image title.
         return self.alt
 
+    # If the image is both large and its height-to-width ratio is approximately
+    # 1/2 we instruct the template to render large Twitter cards
+    # See https://dev.twitter.com/cards/types/summary-large-image
+    @property
+    def summary_large_image(self):
```

Comment:
@richaagarwal What about just `is_large`? So that it'd be `page.meta_image.is_large`?